packages/apps/redis/values.yaml

```
@@ -12,11 +12,11 @@ replicas: 2
 storageClass: ""
 authEnabled: true
 
-## @param resources Resources
+## @param resources Explicit CPU and memory configuration for each Redis replica. When left empty, the preset defined in `resourcesPreset` is applied.
 resources: {}
  # resources:
  #   cpu: 4000m
  #   memory: 4Gi
  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)